CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to {{ project }}
+
+The {{ project }} project team welcomes contributions from the community. Before you start working with {{ project }}, please
+read our [Developer Certificate of Origin](https://cla.vmware.com/dco). All contributions to this repository must be
+signed as described on that page. Your signature certifies that you wrote the patch or have the right to pass it on
+as an open-source patch.
```

Comment:
Since some of our projects use the CLA and some use the DCO, we should probably have 2 sections (one for CLA and one for DCO) and make it clear that they need to delete one of them.